mlf_core.cfg

```diff
@@ -2,7 +2,7 @@
 current_version = 0.1.0-SNAPSHOT
```

Comment:
Let's bump this to 0.1.0  + update the changelog afterwards + then we can also create the release afterwards